sdk/core/src/CommonNetworkMember.ts

```diff
@@ -176,7 +178,9 @@ export abstract class CommonNetworkMember<TOptions extends SdkOptions = SdkOptio
     this._revocationApiService = new RevocationApiService({ revocationUrl, accessApiKey })
     this._userManagementService = new UserManagementService({ clientId, userPoolId, keyStorageUrl, accessApiKey })
     this._keyManagementService = new KeyManagementService({ keyStorageUrl, accessApiKey })
-    this._walletStorageService = new WalletStorageService(keysService, platformEncryptionTools, {
+    const didAuthService = new DidAuthService({ encryptedSeed, encryptionKey: password })
+    this._walletStorageService = new WalletStorageService(keysService, didAuthService, platformEncryptionTools, {
```

Comment:
can we pass the did to **walletStorageService** here to prevent passing it with each method call?